packages/plugins/plugin-outliner/src/types/types.ts

```diff
@@ -48,47 +49,38 @@ export const JournalType = Schema.Struct({
 }).pipe(
   Type.def({
     typename: 'dxos.org/type/Journal',
-    version: '0.1.0',
+    version: '0.2.0',
   }),
 );
 
 export interface JournalType extends Schema.Schema.Type<typeof JournalType> {}
 
+//
+// Helpers
+//
+
+export const createOutline = (name?: string, content?: string): OutlineType => {
```

Comment:
ideally we wouldn't need to abstract this initialization away the defaults could just be specified on the schema, right?